UI/src/view/csvReportsWorkList.js

```diff
@@ -0,0 +1,489 @@
+(function (root) {
+  root.csvReportsWorkList = function () {
+    WorkListView.call(this);
+    var me = this;
+    this.hrefDir = "#work-list/csvReports";
```

Comment:
Can you check if this is being used

Reply:
I think it is by WorkList

Reply:
WorkListView does not have this variable at all and in this csvReportsWorkList this is not being used so I will delete it.